apprise/plugins/NotifyXMPP/__init__.py

```diff
@@ -216,6 +201,7 @@ def __init__(self, targets=None, jid=None, xep=None, **kwargs):
         # By default we send ourselves a message
         if targets:
             self.targets = parse_list(targets)
+            self.targets[0] = self.targets[0][1:]
```

Comment:
@linkmauve : Your code is already merged, but i just stumbled on this now. I'm just curious if you could explain to me why we're removing the first entry from the list of targets and keeping the rest?

Reply:
This is actually the first character of the first entry, which was previously always a `'/'` and would make the XMPP server close our connection.  I’m definitely not sure this was the proper fix, but something somewhere in the parsing of the URI is keeping the initial slash, that is `xmpp://foo:bar@baz/quux@target` would otherwise try to send a message `to='/quux@target'`, which is an obviously invalid JID and the server rightfully closes our connection.

Reply:
Thanks for the reply; I believe I tracked the issue down. As a result i created one more small PR (#487) that _should fix it_ and allow you to delimit as many targets as you want on the URL path (`/target1/target2/targetN/`) . :+1: .

Would you be able to give it a quick test for me?  You're so enriched in this XMPP world where as i don't even have a test client :slightly_smiling_face: 

You should just be able to check the branch out and use the `/bin/apprise` reference from within it (that will allow you to run an apprise instance against the branch you're within)